examples/mongocxx/get_values_from_documents.cpp

```diff
@@ -18,7 +18,22 @@
 #include <mongocxx/instance.hpp>
 #include <mongocxx/client.hpp>
 
-using namespace bsoncxx;
+using bsoncxx::builder::stream::close_array;
+using bsoncxx::builder::stream::finalize;
+using bsoncxx::builder::stream::open_array;
+using bsoncxx::type::k_array;
+using bsoncxx::type::k_document;
+using bsoncxx::type::k_int32;
+using bsoncxx::type::k_int64;
+using bsoncxx::type::k_oid;
+using bsoncxx::type::k_utf8;
+
+using doc_builder = bsoncxx::builder::stream::document;
```

Comment:
Instead of seeing type aliases used for these, I'd much rather either see any of the following, in decreasing order of preference: 1) spelling them out, like the other mongocxx examples do, 2) putting back the mongocxx using-declaration that you had in your first patch set, or 3) using auto.  We should change the names for these classes if we dislike them, but until then I don't like encouraging users to use different names for them.

Not sure if I was unclear in my earlier suggestion, apologies if so.